main/lsp/requests/move_method.cc

```diff
@@ -0,0 +1,224 @@
+#include "main/lsp/requests/move_method.h"
```

Comment:
I'd love to keep `main/lsp/requests/` focused exclusively on the code that handles requests.

What do you think about calling this file something like

```
main/lsp/MoveMethod.cc
main/lsp/MethodCallSiteRenamer.cc
...
```

(flexible on the name, but just up one level in the tree)

Reply:
Sounds good to me! 
Somewhat related question, do we have a code style regarding naming files and variables? I've seen a mix of camel case and snake case for files and variables/functions.